common/src/main/java/com/duckduckgo/app/autofill/JavascriptInjector.kt

```diff
@@ -43,7 +43,10 @@ class FileBasedJavaScriptInjector @Inject constructor() : JavascriptInjector {
 
     override fun getFunctionsJS(): String {
         if (!this::functions.isInitialized) {
-            functions = loadJs("autofill.js")
+            // this can be enabled to see more verbose output from the autofill JS; useful for debugging autofill-related issues
+            val debugMode = false
```

Comment:
Suggestion: We could also align this with AppBuildConfig.isDebug so we allow debug every time it's a debug build.

Reply:
Good idea, and that was my first thought too, but don't want the risk that we'll test everyday against different behaviour than we'd see in prod version. 

Want to test against the important one, and manually switch on extra debugging as a temporary local step.